reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -362,6 +456,55 @@ def download_deeprec_resources(azure_container_url, data_path, remote_resource_n
     os.remove(os.path.join(data_path, remote_resource_name))
 
 
+def mrr_score(y_true, y_score):
```

Comment:
I'm thinking that we should put on the same place these metrics with the metrics that we already have in the repo https://github.com/microsoft/recommenders/blob/master/reco_utils/evaluation/python_evaluation.py. Even though some of them are the same (ie ndcg) the inputs are different, ones are pandas and others are numpy. Maybe we should rename https://github.com/microsoft/recommenders/blob/master/reco_utils/evaluation/python_evaluation.py to `pandas_evaluation.py`, and create a new file names `numpy_evaluation.py` where all these metrics can be added?

What do you guys think @Leavingseason @yueguoguo @anargyri @gramhagen

Reply:
By all means, these metrics would be useful beyond Deeprec, especially MRR which we don't currently have somewhere. I don't think we need to split `python_evaluation.py`, we could have separate functions for numpy, or a data type argument.

Reply:
yes, exactly

Reply:
I am not sure what is the final disign. Can you leave as it is and do the evaluation refactor in another PR?

Reply:
another PR makes sense, added here an issue to track it https://github.com/microsoft/recommenders/issues/1012